app/main/profile-command-handler.js

```diff
@@ -40,16 +45,35 @@ export async function handler(
       .set('recentBookmarks', new Immutable.List(recentBookmarks));
   }
 
+  async function startSession(scope: number, payload: Object = {}): Promise<void> {
+    const { ancestorId } = payload;
+    const sessionId = await storage.startSession(scope, ancestorId);
```

Comment:
Do we want error handling for this or is it fine to assume for now that the storage will never reject?  Looks like that's what we are doing anyway on app startup so it's probably fine.  It's just that the respond callback will never fire which might leave the UI in an odd state